### PR TITLE
Add 2 new Tools and 1 new bridge

### PR DESCRIPTION
**Pull request by Vytek**
# Description

- [algorand.dev] (https://algorand.dev/) - Algorand Developer: The unofficial developer resource. 
- [daotools] (https://daotools.org/) - Manage all your DAO voting sessions in one Dapp.
- [ptokens](https://dapp.ptokens.io/swap?asset=btc&from=btc&to=algorand) - pNetwork Officially Launched Cross-Chain Bridges for Algorand

# Checklist:

_Please make sure you reviewed the checklist and comply with each requirement:_

-   [ x] My code follows the [contribution guidelines](../CONTRIBUTING.md) of this project
-   [ x] This pull request has a title in the format `Add Name of Resource`:
    -   ✅ Add `py-algorand-sdk`
    -   ❌ Update readme.md

> ⚠️ PLEASE NOTE - Do not expect a prompt review for your PR unless you have truthfully went over contribution guidelines, filled the PR description correctly AND most importantly your changes are passing linters in [GitHub Actions pipeline](https://github.com/aorumbayev/awesome-algorand/actions/workflows/ci.yaml).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,7 +255,9 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 - [AgorHash](http://www.agorhash.com/) - Public, permissionless, decentralized and uncensorable free speech protocol.
 - [QRCode Generator](https://github.com/emg110/algorand-qrcode)
 - [algofractals](https://github.com/aorumbayev/algofractals) - Mint randomly generated mandelbrot fractals with embedded ARC69 tags.
-- [algorewards](https://algorewards.github.io/) - Free and unofficial Algorand governance reward calculator. Hosted on GitHub Pages. 
+- [algorewards](https://algorewards.github.io/) - Free and unofficial Algorand governance reward calculator. Hosted on GitHub Pages.
+- [algorand.dev] (https://algorand.dev/) - Algorand Developer: The unofficial developer resource.
```
Comment (aorumbayev) [inline README.md:259]:
```suggestion
- [algorand.dev](https://algorand.dev/) - Algorand Developer: The unofficial developer resource. 
```

File: README.md [hunk 2]
```diff
@@ -254,7 +255,9 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 - [AgorHash](http://www.agorhash.com/) - Public, permissionless, decentralized and uncensorable free speech protocol.
 - [QRCode Generator](https://github.com/emg110/algorand-qrcode)
 - [algofractals](https://github.com/aorumbayev/algofractals) - Mint randomly generated mandelbrot fractals with embedded ARC69 tags.
-- [algorewards](https://algorewards.github.io/) - Free and unofficial Algorand governance reward calculator. Hosted on GitHub Pages. 
+- [algorewards](https://algorewards.github.io/) - Free and unofficial Algorand governance reward calculator. Hosted on GitHub Pages.
+- [algorand.dev] (https://algorand.dev/) - Algorand Developer: The unofficial developer resource. 
+- [daotools] (https://daotools.org/) - Manage all your DAO voting sessions in one Dapp.
```
Comment (aorumbayev) [inline README.md:260]:
```suggestion
- [daotools](https://daotools.org/) - Manage all your DAO voting sessions in one Dapp.
```

File: README.md [hunk 3]
```diff
@@ -220,6 +220,7 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 - [Algomint](https://algomint.io/) - Centralized BTC and ETH bridge to Algorand.
 - [QUIGON](https://bridge.quigon.com/) - Centralized cross-chain NFT bridge.
 - [GlitterFinance](https://glitter.finance/) - Centralized (soon to be decentralized upon TEAL 6) bridge for Solana, Polygon, Terra and Cudos to Algorand.
+- [ptokens](https://dapp.ptokens.io/swap?asset=btc&from=btc&to=algorand) - pNetwork Officially Launched Cross-Chain Bridges for Algorand
```
Comment (aorumbayev) [inline README.md:223]:
```suggestion
- [ptokens](https://dapp.ptokens.io/swap?asset=btc&from=btc&to=algorand) - pNetwork Officially Launched Cross-Chain Bridges for Algorand.
```

**Review by aorumbayev — CHANGES_REQUESTED** [review]
Please address these to fix linter errors, after that we can merge

**Review by Vytek — COMMENTED** [review]
All ok!

**Comment by aorumbayev** [comment]
> All ok!

u sure you commit the changes? Pipeline still failing -> https://github.com/aorumbayev/awesome-algorand/actions/runs/2359388979

**Comment by Vytek** [comment]
Sorry, my error. Now is all ok!

**Review by aorumbayev — APPROVED** [review]

**Pull request merged**